### DISCUSSION TITLE
Copy and Paste Icons Across Nodes

### DISCUSSION THREAD

**Discussion by mathogre** [freeplane/freeplane#313] [Help] [answered]
Hi!

I'm migrating from Freemind to Freeplane.  One of the features from Freemind was the ability to copy and paste icons from one node to another.  It was done via node formatting commands.  When I try to copy and paste a node format in Freeplane, where the original node has an icon, when I paste the format to the new node, only the node color and/or node background color is pasted and the icon is not.

Is there a way to copy and paste icons from one node to another?

Thanks in advance for any insight you might have.

**Comment by macmarrum** [comment, marked as answer]
In Freeplane there is a setting in Preferences that lets you choose which way the copy/paste works.

You can find the setting by using `Command Search (Ctrl+F1)`

![2022-04-12_18,01,07](https://user-images.githubusercontent.com/7197131/163005349-b7b81290-cebe-44e7-870f-d0e26085d9ac.png)

**Reply by mathogre** [reply]
Thank you so much!  Works perfectly.
